jnigen/example/pdfbox_plugin/lib/pdfbox_plugin.dart

```diff
@@ -0,0 +1,5 @@
+/// File merely exporting the generated bindings from lib/src/third_party
+library pdfbox_plugin;
+
+export 'src/third_party/pdmodel.dart';
```

Comment:
Should we generate a `src/third_party/_all.dart` from jnigen?

That way we don't have to deal with manually adding extra exports here when we add classes.